frontend/angular.json

```diff
@@ -1,126 +1,143 @@
 {
-    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
-    "cli": {
-        "analytics": false,
-        "schematicCollections": ["@angular-eslint/schematics"]
-    },
-    "version": 1,
-    "newProjectRoot": "projects",
-    "projects": {
-        "leetwars": {
-            "projectType": "application",
-            "schematics": {
-                "@schematics/angular:component": {
-                    "style": "sass"
-                },
-                "@schematics/angular:application": {
-                    "strict": true
-                },
-                "@schematics/angular:class": {
-                    "skipTests": true
-                },
-                "@schematics/angular:directive": {
-                    "skipTests": true
-                },
-                "@schematics/angular:guard": {
-                    "skipTests": true
-                },
-                "@schematics/angular:pipe": {
-                    "skipTests": true
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false,
+    "schematicCollections": [
```

Comment:
toooo many changes, can you some how add only your changes?

Reply:
It was only one command

![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/106184656/1fcf0a8e-8b86-47c0-8c55-4eb9f2a0f098)